notify/impl.go

```diff
@@ -153,20 +155,24 @@ func (w *Webhook) Notify(ctx context.Context, alerts ...*types.Alert) error {
 
 	var buf bytes.Buffer
 	if err := json.NewEncoder(&buf).Encode(msg); err != nil {
-		return err
+		return false, err
 	}
 
 	resp, err := ctxhttp.Post(ctx, http.DefaultClient, w.URL, contentTypeJSON, &buf)
 	if err != nil {
-		return err
+		return false, err
 	}
 	resp.Body.Close()
 
-	if resp.StatusCode/100 != 2 {
-		return fmt.Errorf("unexpected status code %v from %s", resp.StatusCode, w.URL)
+	if resp.StatusCode/100 == 5 {
+		return true, fmt.Errorf("unexpected status code %v from %s", resp.StatusCode, w.URL)
+	}
+
+	if resp.StatusCode/100 != 2 && resp.StatusCode/100 != 3 {
```

Comment:
yes, of course .. missed to remove that one, it was one of my first thoughts and removed it in all the other ones..